lib/ClientBase.js

```diff
@@ -242,6 +242,15 @@ ClientBase.prototype._getAllHttp = function(opts, callback, headers) {
     }
   }
 
+  if (path && path.includes('transactions')) {
+    // adding this query param to get the nested `fees` 
+    // for the `buy` and `sells` object
+    if (!args) {
```

Comment:
can be `null`
https://github.com/cryptotaxcalculator/coinbase-node/blob/33212ff347360574ad1729ef8d000b8af72776ef/lib/ClientBase.js#L226-L228